mockdata/api/clusters_mgmt/v1/clusters/21696abb0dbkvh0mh4lranlmvvqb11lg/node_pools.json

```diff
@@ -0,0 +1,56 @@
+{
+  "kind": "NodePoolList",
+  "href": "/api/clusters_mgmt/v1/clusters/21696abb0dbkvh0mh4lranlmvvqb11lg/node_pools",
+  "page": 1,
+  "size": 2,
+  "total": 2,
+  "items": [
+    {
+      "kind": "NodePool",
+      "href": "/api/clusters_mgmt/v1/clusters/21696abb0dbkvh0mh4lranlmvvqb11lg/node_pools/21697vs8sunmf4j6jem5f7k5j7ct08ue",
+      "id": "21697vs8sunmf4j6jem5f7k5j7ct08ue",
+      "replicas": 2,
+      "auto_repair": true,
+      "aws_node_pool": {
+        "instance_type": "m5.xlarge",
+        "instance_profile": "production-21696abb0dbkvh0mh4lranlmvvqb11lg-hypermaupadhy-worker",
+        "tags": {
+          "api.openshift.com/environment": "production",
+          "api.openshift.com/id": "21696abb0dbkvh0mh4lranlmvvqb11lg",
+          "api.openshift.com/legal-entity-id": "2KBain48N90sMZ8zhjcHDcILVUw",
+          "api.openshift.com/name": "hypermaupadhy",
+          "api.openshift.com/nodepool": "hypermaupadhy-workers",
+          "dns": "external",
+          "red-hat-clustertype": "rosa",
+          "red-hat-managed": "true"
+        }
+      },
+      "availability_zone": "us-east-1d",
+      "subnet": "subnet-0da631ca7d1b2ef9c",
+      "imageType": "Windows"
```

Comment:
This addition propagates a Machine Type which is Windows LI compatible, meaning it can enable Windows LI (as agreed with the backend team)